graph/client/src/app/feature-tasks/tasks-sidebar.tsx

```diff
@@ -40,10 +40,21 @@ export function TasksSidebar() {
 
   const selectedTarget = params['selectedTarget'] ?? targets[0];
 
-  const [selectedProjects, setSelectedProjects] = useState<string[]>([]);
-
   const currentRoute = useCurrentPath();
-  const routeContructor = useRouteConstructor();
```

Comment:
Did you run into problems with the route constructor? It's meant to make things easier for constructing routes since the routes between dev and release are different. Dev routes need to include the project id while that's not in the release routes.

Reply:
It is up above. I called it `createRoute` :eyes: